tools/shared/src/main/scala/org/scalajs/core/tools/javascript/JSDesugaring.scala

```diff
@@ -1576,8 +1576,24 @@ private[javascript] object JSDesugaring {
           }
 
         case ApplyStatically(receiver, cls, method, args) =>
-          val fun = encodeClassVar(cls.className).prototype DOT method
-          js.Apply(fun DOT "call", (receiver :: args) map transformExpr)
+          val className = cls.className
+          val transformedArgs = (receiver :: args) map transformExpr
+
+          if (classEmitter.isInterface(className)) {
+            val Ident(methodName, origName) = method
+            if (isStrongMode) {
+              js.Apply(js.DotSelect(envField("c", className),
+                  js.Ident("$f_" + methodName, origName)(method.pos)),
+                  transformedArgs)
```

Comment:
Why do we put this in the static namespace of the class in strong mode? Just because?
It seems that in strong mode we now have another collision domain here (default implementation / static method) that we do not have in non-strong mode.